addons/mail/static/src/core/web/thread_service_patch.js

```diff
@@ -55,29 +54,6 @@ patch(ThreadService.prototype, {
         }
         return result;
     },
-    getThread(model, id) {
-        let thread = this.store.Thread.get({ model, id });
-        if (thread) {
-            if (id === false) {
-                return thread;
-            }
-            // to force a reload
-            thread.status = "new";
-        }
-        thread = this.store.Thread.insert({ id, model, type: "chatter" });
```

Comment:
Isn't type = chatter gone if you replace this method by just insert (id, model)?

Reply:
No, because of the `compute()` on `type` field.